packages/orchestration/src/typeGuards.js

```diff
@@ -96,11 +94,13 @@ export const CosmosChainInfoShape = M.splitRecord(
   },
 );
 
-// FIXME more validation
-export const ChainInfoShape = M.any();
+/** @type {TypedPattern<ChainInfo>} */
+export const ChainInfoShape = M.splitRecord({
+  chainId: M.string(),
```

Comment:
Maybe out of scope, but consider including a `CosmosChainInfoShape` for the second argument here. It seems we might be expecting a certain shape for `IBCConnectionInfo`.

`icqEnabled: M.boolean()` and `stakingTokens: M.arrayOf({ denom: M.string() })` seem straight forward to incldue as well.

Reply:
Out of scope and I think it won't work